signup/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓</title>
+  <link rel="stylesheet" href="style.css">
```

Comment:
현재 상대경로를 통해 스타일 시트를 적용하고 있습니다. 절대 경로를 사용하면 변경에 더 단단한 코드를 작성할 수 있어요!
```suggestion
  <link rel="stylesheet" href="/signup/style.css">
```

Reply:
로컬에서 VSCode live server로 작업하고, 마지막에 Netlify에 배포해서 절대경로를 가지게 됩니다. 절대경로를 사용하면 작업 중에는 반영되는 것을 볼 수 없는데 어떻게 하면 될까요? 배포 후에 모든 파일의 경로를 다 절대로 수정하는 것도 힘듭니다.

Reply:
`절대경로를 사용하면 작업 중에는 반영되는 것을 볼 수 없는데` 이 부분이 잘 이해가 가지 않는데요. 절대경로를 사용해도 작업물을 실시간으로 보는데에는 문제가 없는 것이 정상입니다. 조금 더 자세히 설명해 주실 수 있을까요?